### DISCUSSION TITLE
7113: Latest supported firmware should be 60.04.68, not 90.04.86.

### DISCUSSION THREAD

**Discussion by dreirund** [Freetz-NG/freetz-ng#1038] [Allgemein] [answered]
Ahoj,

after I managed to [successfully build Freetz-NG for Fritz!Box 7113](https://github.com/Freetz-NG/freetz-ng/discussions/1037#discussioncomment-10594524), I noticed that the finished image is based on AVM firmare version 90.04.86:

`make` output:  
```
[...]
STEP 3: PACK/SIGN
[...]
source firmware: 7113-AnnexA_de-es-fr-it 90.04.86 rev27557 {INT} (19.02.2014 15:08:23) 
```

That image does not work, it puts the box into a boot loop.

I already tried [that official image from the AVM download server](https://download.avm.de/archive/fritz.box/fritzbox.fon_wlan_7113/firmware/english/FRITZ.Box_Fon_WLAN_7113.de-en-es-it-fr.90.04.86.image), and both AVMs image as well as the one build by Freetz-NG put the box into a boot loop.

According to [router-faq.de](https://www.router-faq.de/?id=fbinfo&hwf=fbfonwlan7113#fbfonwlan7113), the latest supported firmware version is [60.04.68](https://download.avm.de/archive/fritz.box/fritzbox.fon_wlan_7113/firmware/deutsch/FRITZ.Box_Fon_WLAN_7113.60.04.68.image), and flashing that official image works. I do not know what AVM has messed up here.

But I recommend changing Freetz-NG to use [60.04.68](https://download.avm.de/archive/fritz.box/fritzbox.fon_wlan_7113/firmware/deutsch/FRITZ.Box_Fon_WLAN_7113.60.04.68.image for 7113.

Regards!

**Comment by dreirund** [comment]
OK, when I change in `make menuconfig` the "Target region" from "avme - international" (`CONFIG_FREETZ_TYPE_LANG_EN=y`) to "avm - germany" (`CONFIG_FREETZ_TYPE_LANG_DE=y`), firmware 60.04.68 is used. So there might be hardware differences between an "international" and a "German" version(?).

This seems to solve this issue.

Regards!

**Comment by fda77** [comment, marked as answer]
If you flash the wrong firmware, devices does not start anymore ;-)
See https://freetz-ng.github.io "FIRMWARES", the "60" and "90" indicates different hardware
But you could select 7170 and then "alien"

**Reply by dreirund** [reply]
> *But you could select 7170 and then "alien"*

Would this give a newer firmware than 60.04.68 _and_ work with 7113 (german edition)?

And what is "Alien" firmware in general? Is that using a firmware designed for another device (in that case for 7170)?

[FIRMWARES.html](https://freetz-ng.github.io/freetz-ng/FIRMWARES.html#fritzbox-fon-wlan-71xx) shows me that the latest 7170 firmware is 58.04.85 for the `{INT}` variant, and even lower for all other variants, so it seems to be older than 60.04.68 for 7113.

**Comment by fda77** [comment]
I dont know which device you have, german or international?

> should be 60.04.68, not 90.04.86

HWR 60. is german

Even the FOS version is a little smaller, it is possible the alien has more features - but i cant remember which

**Comment by dreirund** [comment]
On Tue, 10 Sep 2024 07:07:57 -0700, fda77 ***@***.***>
wrote:

> I dont know which device you have, german or international?

It seems to be German since with the other firmware the box ended in a
boot loop.

> Even the FOS version is a little smaller, it is possible the alien
> has more features - but i cant remember which

Actually, I tried the 7170 alien for 7113, and the image easily exceeds
the storage.

I keep the 7113.

Only feature I miss is to be used as WLAN client -- which would enable
to use analogue phones as SIP-client within a WiFi with that small box.

**Reply by fda77** [reply]
its called WDS

**Reply by dreirund** [reply]
> *WDS*

Which is a different feature than acting as a Wlan client.

Anyway, I just keep it as it is. I succeded to build an 7170 alien firmare, but it boot-loops. And I doubt that it can work in client mode either.
